tool/plugin/lib/plugin.dart

```diff
@@ -442,13 +442,13 @@ abstract class BuildCommand extends ProductCommand {
           spec.buildForMaster();
         }
 
-        result = await spec.artifacts.provision(
+        /*result = await spec.artifacts.provision(
```

Comment:
Can this be deleted? And same for below

Reply:
Oops, I forgot to clean up the code. Lots to delete.